README.md

```diff
@@ -1757,7 +1757,12 @@ However, this section covers some basics of what to expect with offers, and tips
 
 ### Offers
 
-🔹 Companies can and should work hard to ensure that all candidates are [given equal treatment](https://rework.withgoogle.com/guides/pay-equity/steps/introduction/) in the hiring process. But candidates should also know their worth and understand that they can negotiate to get better offers.
+🔹 Companies can and should work hard to ensure that all candidates are
+[given equal treatment](https://rework.withgoogle.com/guides/pay-equity/steps/introduction/)
+in the hiring process but
+[inequalities persist](https://iwpr.org/publications/gender-wage-gap-2017-race-ethnicity/).
```

Comment:
This is such a complex issue, we ideally should add 3-5 links on with thoughtful overviews of current facts:

Dimensions include
- gender and race disparities on pay broadly in the US (that link is good for the former; here is another one covering both racial and gender disparities:
http://www.pewresearch.org/fact-tank/2016/07/01/racial-gender-wage-gaps-persist-in-u-s-despite-some-progress/ )
- executive pay disparities and lack of diversity
- tech industry pay disparities and lack of diversity 

Tons of things written on the latter two; the hard part is selecting a few good overviews, with statistics.

Reply:
Also, I do think it would be appropriate to link to popular and thoughtful, opinionated writings on this topic as well, but doing that in a thorough yet brief way is a tall order and a separate PR.

Reply:
Okay, so for this PR I will rework text and add links representing each dimension but not attempt to survey the range of opinionated thought on the topic.